packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -70,13 +74,28 @@ function CreateIndexModal({
     undefined
   );
 
+  // @experiment Early Journey Indexes Guidance & Awareness  | Jira Epic: CLOUDP-239367
+  const enableInIndexesGuidanceExp = usePreference('enableIndexesGuidanceExp');
+  const showIndexesGuidanceVariant =
+    usePreference('showIndexesGuidanceVariant') && enableInIndexesGuidanceExp;
+
+  useFireExperimentViewed({
+    testName: TestName.earlyJourneyIndexesGuidance,
+    shouldFire: enableInIndexesGuidanceExp && isVisible,
```

Comment:
Maybe I'm understanding this wrong, should this be `showIndexesGuidanceVariant` and not `enableInIndexesGuidanceExp`?
```suggestion
    shouldFire: showIndexesGuidanceVariant && isVisible,
```

Reply:
nope this should fire for when user is in the control as well so it should be `enableInIndexesGuidanceExp`

Reply:
Should we track that they're in the control then? Or is there already a way we differentiate that they are shown/not shown the new experience?

Reply:
this tracks if they are in the control as well, if they are in the experiment `enableInIndexesGuidanceExp` is true. i need to confirm once i get the latest into our mms but i think by default we have code set up to send down whether user is in the variant or control

Reply:
Would it make things easier if we track if they're in the control or shown the experience here as some metadata for the tracking event? If it's on the mms side I suppose we don't need to since we'll be able to correlate them anyway. Feel free to resolve!